cmd/ronin/config.go

```diff
@@ -152,6 +153,14 @@ func makeConfigNode(ctx *cli.Context) (*node.Node, gethConfig) {
 	}
 	applyMetricConfig(ctx, &cfg)
 
+	// setup mock config
+	if ctx.GlobalIsSet(utils.MockValidatorsFlag.Name) && ctx.GlobalIsSet(utils.MockBlsPublicKeysFlag.Name) {
+		consortiumCommon.Validators, err = consortiumCommon.NewMockValidators(ctx.GlobalString(utils.MockValidatorsFlag.Name), ctx.GlobalString(utils.MockBlsPublicKeysFlag.Name))
```

Comment:
That means we need to have a function to get validators? such as `GetValidators()`?
In snapshot, `MockContract` cannot be accessed via consortium. Therefore, I need to set it as a global variable, otherwise, we need to add a method to get it

Reply:
Okay, I got the reason why Validators needs to be public, but I think we can move the consortiumCommon.Validators assignment into the consortiumCommon.NewMockValidators